lib/Compiler.php

```diff
@@ -10,7 +10,6 @@
 namespace PHPCompiler;
 
 // This is used as a property type, phan is confused.
```

Comment:
```suggestion
```

Just realized I could do my comment as a github suggestion..

Reply:
Never used the github suggestions to make a commit, so that was a first.